Sources/Search.php

```diff
@@ -508,11 +507,10 @@ function PlushSearch2()
 	{
 		$request = $smcFunc['db_query']('', '
 			SELECT b.id_board
-			FROM {db_prefix}topics AS t
-				INNER JOIN {db_prefix}boards AS b ON (b.id_board = t.id_board)
-			WHERE t.id_topic = {int:search_topic_id}
+			FROM {db_prefix}topics AS b
```

Comment:
Once `query_see_topic_board` is added, the query here can go back to using `{db_prefix}topics AS t` along with `{query_see_topic_board}`